### PR TITLE
addresses an issue found in 10.6.0 QA

### PR DESCRIPTION
**Pull request by segun**
Fixes: An issue found in 10.6.0 QA

Explanation:  

Token approval full url under permission tab, [ScreenCap](https://drive.google.com/open?id=1X6v_vkwzZDQQU6iGfjb34GqhnrQjmbl0).
 Design implementation has the capitalized domain name, [Figma](https://www.figma.com/file/eEFSq73S1NLEReogz1WTUG/ERC20-token-approval?node-id=219%3A536).

  - 
  - 
  -

### CODE REVIEW

**Review by danjm — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -2875,7 +2875,7 @@
     "message": "Transaction encountered an error."
   },
   "transactionFee": {
-    "message": "Transaction Fee"
+    "message": "Transaction fee"
```
Comment (tmashuang) [inline app/_locales/en/messages.json:2878]: `yarn test:unit:jest -u` is going to needed to be ran, and the changes committed, to update one of the swaps snapshots tests with this change.

Comment (tmashuang) [reply app/_locales/en/messages.json:2878]: Also, the other tests this changes that need to be addressed are 
https://github.com/MetaMask/metamask-extension/blob/8ca0d24f8251ecc6ab915ebaae6929b060d07b5f/ui/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.test.js#L22
and
https://github.com/MetaMask/metamask-extension/blob/95e3be4a85fdfc328e9365811f5972d99c7c0e48/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.test.js#L61

**Comment by tmashuang** [comment]
yarn test:unit:jest -u is going to needed to be ran, and the changes committed, to update one of the swaps snapshots tests with this change.

Also, the label changes will need to get updated in the test file.
[confirm-approve-content.component.test.js](https://github.com/MetaMask/metamask-extension/blob/95e3be4a85fdfc328e9365811f5972d99c7c0e48/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.test.js)

**Comment by segun** [comment]
> `yarn test:unit:jest -u` is going to needed to be ran, and the changes committed, to update one of the swaps snapshots tests with this change.
> 
> Also, the label changes will need to get updated in the test file. [confirm-approve-content.component.test.js](https://github.com/MetaMask/metamask-extension/blob/95e3be4a85fdfc328e9365811f5972d99c7c0e48/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.test.js) and
> 
> https://github.com/MetaMask/metamask-extension/blob/8ca0d24f8251ecc6ab915ebaae6929b060d07b5f/ui/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.test.js#L22

Thank you.

**Comment by tmashuang** [comment]
More label changes need to be addressed in the confirm-approve-content.component.test.js
https://github.com/MetaMask/metamask-extension/blob/95e3be4a85fdfc328e9365811f5972d99c7c0e48/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.test.js#L75-L82

**Review by danjm — APPROVED** [review]

**Review by tmashuang — APPROVED** [review]

**Pull request merged**